app/scripts/analysis/GetData/GetDataMainCtrl.controller.coffee

```diff
@@ -207,7 +275,7 @@ module.exports = class GetDataMainCtrl extends BaseCtrl
       throw err
     )
 
-  getSocrData: ->
+  getSocrDataset: ->
```

Comment:
@chartotu19 at least this change is lost if this branch is updated from current state of `dev`